### PR TITLE
Added the brand property to the StripeCardData and fix style issues, lint

### PR DESCRIPTION
**Pull request by aluanhaddad**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `npm run lint -- package-name` if a `tslint.json` is present.<sup>1</sup>

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://stripe.com/docs/api#card_object
- [x] Increase the version number in the header if appropriate.<sup>2</sup>

Unsure about ambient declaration:
Removed as it seemed to be in error:
  - Letter casing was not correct. (examining source suggests the global may be re-exported as an AMD module but under the name `"stipe"`, not `"Stripe"`
  - The export was the `interface` StipeStatic which is only useful to refer to the type of the global defined by the module (the global `Stripe` created by `stripe.js` from `https://js.stripe.com/v2/`)
  - This module does not seem a suitable candidate for a UMD declaration.

Interface change
Added the `brand` property to the `StripeCardData` declaration with the type
`'Visa' | 'American Express' | 'MasterCard' | 'Discover JCB' | 'Diners Club' | 'Unknown'`

**Notes**:
  1. See the suppression in tslint.json (added) and then 
  2. The stripe client side APIs are only not published as dedicated packages for compliance reasons. A release date versioning schema is used.

### CODE REVIEW

**Comment by a904guy** [comment]
:+1:

**Comment by aluanhaddad** [comment]
Is there anything I need to do from here?

**Comment by a904guy** [comment]
:+1:

**Comment by amritk** [comment]
👍

File: stripe/tslint.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "../tslint.json",
+    "rules": {
+        // The Stripe client side APIs are not made available to package managers for direct installation.
+        // As explained compliance reasons. Source: https://github.com/stripe/stripe-node/blob/master/README.md#these-are-serverside-bindings-only
+        // A release date versioning schema is used to version these APIs. It is not compatable with semver.
+        "dt-header": false
```
Comment (andy-ms) [inline stripe/tslint.json]: You could just use `0.0` in the header and include a comment somewhere in the source.

**Comment by andy-ms** [comment]
The definitions of the global variable and module may have been wrong before, but if you delete them both it seems like there's no way to actually use this package?

File: stripe/tslint.json (hunk 1 above)
Comment (aluanhaddad) [inline stripe/tslint.json]: Done!

**Comment by aluanhaddad** [comment]
@andy-ms 


> The definitions of the global variable and module may have been wrong before, but if you delete them both it seems like there's no way to actually use this package?

If it is installed, everything is available in the global namespace. What would make it unusable?

**Comment by andy-ms** [comment]
If you remove `tslint.json` it won't be linted at all.

**Comment by aluanhaddad** [comment]
@andy-ms Thank you for explaining. I appreciate it.

**Pull request merged**